src/views/AgeGroup/index.js

```diff
@@ -187,7 +188,11 @@ const AgeGroup = () => {
     user.loggedIn && categories
       ? categories.map(({ groups }) => {
           groups.map((group) => {
```

Comment:
Tähän map funktioon on kertynyt aikamoinen `if else` rakennelma. Tätä vois yrittää jossain vaihessa miettiä paremmaks. Vois vaikka laittaa jonkun `// TODO: Find a way to simplify this if else`